### ISSUE TITLE
Doesn't seem to support deps with vanity URLs

### ISSUE THREAD

**Issue by mrobinsn** [ribice/glice#4]
I have a project which imports `go.uber.org/atomic` (points to https://github.com/uber-go/atomic).

`glice` does not pick up a License for this project even though it has an MIT license in the repo.

I suspect this is because it isn't following the vanity URL metadata redirect.

```
$ curl -v https://go.uber.org/atomic

<!DOCTYPE html>
<html>
    <head>
        <meta name="go-import" content="go.uber.org/atomic git https://github.com/uber-go/atomic">
        <meta name="go-source" content="go.uber.org/atomic https://github.com/uber-go/atomic https://github.com/uber-go/atomic/tree/master{/dir} https://github.com/uber-go/atomic/tree/master{/dir}/{file}#L{line}">
        <meta http-equiv="refresh" content="0; url=https://godoc.org/go.uber.org/atomic">
    </head>
    <body>
        Nothing to see here. Please <a href="https://godoc.org/go.uber.org/atomic">move along</a>.
    </body>
</html>
```

**Comment by ribice** [comment]
You're correct. I plan to rewrite `glice` in following months and include this as a feature.

**Comment by ribice** [comment]
Resolved with #11
